independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -205,4 +256,53 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
         }
         return ElementCatalogBuilder.getMembersForElements(ec, eKeys);
     }
+
+    private void addModuleToParentPom(QuarkusCommandInvocation invocation, Model parent,
```

Comment:
just a thought - parent pom is not necessarily the same as the aggregator pom.
you can easiliy have a parent that is not in your parent directory and in parent directory is only an aggregator.

I'm wondering if it would be worth printing a info or warning to users in case of the parent dir found pom has <modules> but no build sections then most likely user want to control if we generated the right parent pom....

technically maybe the only thing we should do is to add our selves to the aggregator pom ?